log/log.go

```diff
@@ -55,6 +58,7 @@ func CleanExit(format string, v ...interface{}) {
 func Err(format string, v ...interface{}) {
 	fmt.Fprint(Stderr, pretty.Colorize("{{.Red}}[ERROR]{{.Default}} "))
 	fmt.Fprintf(Stderr, appendNewLine(format), v...)
+	ErrorState = true
```

Comment:
I was asking for a comment as a defensive measure for future maintainability. Anyway, not a showstopper for this PR.

I may go through portions of the codebase and add concurrency related comments or make concurrency safe in a later PR. There's little concurrency (if any) in this codebase right now that I know of, so I'm not considering it high priority.